web-api/terraform/template/elasticsearch.tf

```diff
@@ -15,4 +15,10 @@ resource "aws_elasticsearch_domain" "efcms-search" {
   snapshot_options {
     automated_snapshot_start_hour = 23
   }
+
+  log_publishing_options = {
+    enabled                  = "true"
+    cloudwatch_log_group_arn = "arn:aws:logs:${var.aws_region}:${data.aws_caller_identity.current.account_id}:log-group:/aws/aes/debug_${var.environment}"
```

Comment:
Re: your comment about this ARN and being too prescriptive — an easy way to make sure we’re using the right ARN would be to create the log group and reference it: 

```hcl
resource "aws_cloudwatch_log_group" "elasticsearch_application_logs" {
  name = "/aws/aes/debug_${var.environment}"
}

resource "aws_elasticsearch_domain" "efcms-search" { 
  …
  log_publishing_options {
    enabled = "true"
    cloudwatch_log_group_arn = aws_cloudwatch_log_group.elasticsearch_application_logs.arn
  }
}
```